rrpproxy/utils/populate_domain_list.py

```diff
@@ -0,0 +1,24 @@
+
```

Comment:
You could consider using the `pandas` library like suggested here https://stackoverflow.com/a/50113635 I'v used it before and quite liked it. Not sure if it completely suits your needs here.

Reply:
adding pandas as a dependency might be kind of heavy just for that dict transformation

Reply:
Yeah, would agree in this case.